lib/engine.js

```diff
@@ -57,8 +57,7 @@ const runItInternal = (answers, directory, timeout) => {
   try {
     if (answers.wfType === 'Standalone') {
       if (answers.wfStandaloneCount > 1) {
-        const template = answers.hawkAgent === 'javaAgent' ? 'hawkfly-new' : 'hawkfly-old';
-        dc.scale(directory, template, answers.wfStandaloneCount);
+        dc.scale(directory, 'hawkfly', answers.wfStandaloneCount);
```

Comment:
right, the service in the both yaml configs is called `hawkfly`, thanks for catching this